src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,21 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
+    public static final String EXAMPLE = "123, some street, some unit, some postal code";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String ADDRESS_SPLIT_REGEX = ",";
+    public static final int BLOCK_ADDRESS_INDEX = 0;
```

Comment:
2 things 
- Good to see you have eliminated magic numbers in the code
- However, please take a look at the coding standards and find out what is the correct convention for naming constants